VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -64,6 +64,13 @@
                 "ajidxCSV"       : NTupleCollection("aJCidx",    objectInt, 2,help="additional jet indices CSV"),
                 "hjidx3cj"       : NTupleCollection("hJ3Cidx",    objectInt, 2,help="Higgs jet indices 3 cen jets"),
                 "ajidx3cj"       : NTupleCollection("aJ3Cidx",    objectInt, 2,help="additional jet indices 3 cen  jets"),
+
+                # Boosted analysis objects
+                "ungroomedFatjets"       : NTupleCollection("ungroomedFatjets", fatjetType, 10,help="CA, R=1.5, pT > 200 GeV, no grooming"),
```

Comment:
you should add those only when running a config that has the boost stuff in (otherwise it crashes)